omero/training-server/playbook.yml

```diff
@@ -63,11 +63,26 @@
       become: yes
       pip:
         name:
-        - "omero-figure==3.1.2"
-        - "omero-fpbioimage==0.2.0"
-        - "omero-webtagging-autotag==3.0.2"
-        - "omero-webtagging-tagsearch==3.0.3"
-        - "omero-iviewer==0.4.1"
+        - "omero-figure=={{ omero_figure_tag }}"
+        - "omero-fpbioimage=={{ omero_fpbioimage_tag }}"
+        - "omero-mapr=={{ omero_mapr }}"
+        - "omero-parade=={{ omero_parade }}"
+        - "omero-webtagging-autotag=={{ omero_webtagging_autotag }}"
+        - "omero-webtagging-tagsearch=={{ omero_webtagging_tagsearch }}"
+        editable: False
+        state: present
+        # variable comes from role openmicroscopy.omero-web
+        virtualenv: "{{ omero_web_basedir }}/venv"
+        virtualenv_site_packages: yes
+      notify:
+        - restart omero-web
+
+    - name: Omero.web plugins testpypi | plugin install via testpypi
+      become: yes
+      pip:
+        extra_args: -i https://testpypi.python.org/pypi
```

Comment:
I might have missed a discussion here but why are we installing a version of OMERO.iviewer from test Pypi?

Reply:
It was before when an rc was tested.
testpypi will no longer be used as discussed last week (you were present)

Reply:
Thanks for the clarification, this is what I remembered. Generally, :+1: for only consuming PyPI packages in our production playbooks. In the cases where we need to use milestones or rcs, we should be able to create [pre-releases](https://www.python.org/dev/peps/pep-0440/#pre-releases) as necessary.